README.md

```diff
@@ -378,6 +378,59 @@ the ownership voucher and ownership certificate.
     we have validated the identity and integrity of the device and its
     software components. It is ready to serve traffic.
 
+### A Note on How To Secure The First gNSI.Certz Connection After Successful Attestation
+
+In some deployment environment, an user might prefer to install production CA
+issued cert ONLY if the device has passed the attestation. iDevID cert is a good
```

Comment:
first introduction problems stink :(

Reply:
* Technically, we don't have to use oDevID for the first gNSI.Certz. 

* Waiting for vendor's comment/feedback about TPM keys.

Reply:
I updated `attestz` repo accordingly in https://github.com/openconfig/attestz/pull/21. As discussed, please also update `bootz` and `certz` repos to reflect the following:

* A device is shipped to a switch owner with a default SSL profile configured to rely on the IDevID key pair and IDevID TLS cert (signed by switch vendor CA) for all RPCs.
* A device will rely on IDevID cert by default until either the default SSL profile is updated (during initial `enrollz`, for example) or another SSL profile is set (during `certz`, for example).
* Once device receives a new Owner IDevID cert (signed by switch owner CA) in [RotateOIakCert](https://github.com/openconfig/attestz/blob/main/proto/tpm_enrollz.proto#L95) RPC, the device must update its default SSL profile to rely on Owner IDevID cert instead of IDevID cert. The device must then force all its gRPC servers to only rely on the Owner IDevID cert.